pkg/FontChooser/app/app.js

```diff
@@ -48,13 +49,33 @@ const bootSystem = async (system, recipe, {fontData, suggested}) => {
   if (recipe.$stores.suggested) {
     recipe.$stores.suggested.$value = suggested;
   }
+  const policy = await fetch('./policy.json').then(t => t.text());
```

Comment:
nit thoughts, can do later:
- maybe move the ibis logic chunk(s) into a `validatePolicy()` function or something for readability/maintainability
- error handling even if just logging probably worth improving (use logger vs console, use error/warning where appropriate, etc.)
- how large are the files, how long does the load and validate take to execute, asking re: whether we might want to do different parts at different times (we could postpone validate until user actually clicks the button, for example). I am just curious to get a sense of data size and timing (I can try it myself once it's merged too).